### DISCUSSION TITLE
Stream not being published from VLC?

### DISCUSSION THREAD

**Discussion by PulseBeat02** [bluenviron/mediamtx#1072] [Questions]
## Which version are you using?

v0.19.2

## Which operating system are you using?

- [ ] Linux amd64 standard
- [ ] Linux amd64 Docker
- [ ] Linux arm64 standard
- [ ] Linux arm64 Docker
- [ ] Linux arm7 standard
- [ ] Linux arm7 Docker
- [ ] Linux arm6 standard
- [ ] Linux arm6 Docker
- [X] Windows amd64 standard
- [ ] Windows amd64 Docker (WSL backend)
- [ ] macOS amd64 standard
- [ ] macOS amd64 Docker
- [ ] Other (please describe)

## Describe the issue

I first start a stream in VLC Media Player and set the port to be the RTSP port (that was specified in the configuration). I tried all ports that were listed and streamed to it using VLC but the server would not be able to find it (there was also no output in console). I also streamed on my local PC (localhost).

## Describe how to replicate the issue

1. start the server
2. Open VLC Media Player and create an RTSP stream. Then input your URL and set the address to be localhost and port to be the specified port in the RTSP server.
3. No result is shown in console for the RTSP server. Also visiting the URL does not make a valid request.

## Did you attach the server logs?

no

## Did you attach a network dump?

no

Note: the server is able to accept streams from FFmpeg but not from VLC Media Player for some reason.

**Comment by aler9** [comment]
Hello, VLC doesn't support publishing to a RTSP server, but it creates its own RTSP server from which the stream can be played or proxied.

If you want to "publish" a stream from VLC and distribute it through rtsp-simple-server, you can do the following:
1. menu "Media", "Stream", pick your source, click the "stream" button", "new destination", "RTSP", "Add", pick port 8556 (or a port that is not used by rtsp-simple-server) path "/stream", button "next", "next", "stream"
2. use this rtsp-simple-server configuration:

   ```yml
   paths:
     vlc:
       source: rtsp://127.0.0.1:8556/stream
   ```

3. start rtsp-simple-server
4. now you can read the stream by using the URL `rtsp://127.0.0.1:8554/proxied` (if you want to pass through rtsp-simple-server), or the URL `rtsp://127.0.0.1:8556/stream` (if you want to skip rtsp-simple-server and use directly VLC)

[a comment by an automated account is not shown]
